src-vue/src/views/DeveloperView.vue

```diff
@@ -152,6 +210,42 @@ export default defineComponent({
                     });
                 });
         },
+        async getTags() {
+            await invoke<TagWrapper[]>("get_list_of_tags")
+                .then((message) => {
+                    this.ns_release_tags = message;
+                    ElNotification({
```

Comment:
These empty notifications feel weird.
Maybe you can put a success message there?